docs/administration/projects/project-motd.md

```diff
@@ -5,21 +5,24 @@ The motd content is interpreted as [markdown](http://commonmark.org/help/) text
 
```

Comment:
If we are going to make changes let's improve this introduction.  "MOTD" should be described as "Message Of The Day" before using the acronym everywhere else.  Improve context of why and how it's used, etc.